brvs-rules/src/main/java/iroha/validation/rules/impl/whitelist/CheckWhitelistRule.java

```diff
@@ -0,0 +1,93 @@
+package iroha.validation.rules.impl.whitelist;
+
+import com.google.common.base.Strings;
+import iroha.protocol.Commands;
+import iroha.protocol.TransactionOuterClass;
+import iroha.validation.rules.Rule;
+import java.security.KeyPair;
+import java.util.Map;
+import java.util.Objects;
+import jp.co.soramitsu.iroha.java.IrohaAPI;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Rule that checks whitelisted addresses on withdrawal.
+ */
+public class CheckWhitelistRule implements Rule {
+
+  private static final Logger logger = LoggerFactory.getLogger(UpdateWhitelistRule.class);
+
+  private static final String withdrawalAccount = "notary@notary";
```

Comment:
But I send money to a different account.

Reply:
Moved to context